physics/kinematic_equations/kinematic_equations.py

```diff
@@ -0,0 +1,27 @@
+# assuming that acceleration is constant
+
+# v -> velocity (m/s)
+# v_0 -> initial velocity (m/s)
+# a -> acceleration (m/s/s)
+# delta_x -> change in position (m)
+# t -> time (s)
+
+# if we don't know delta_x
+def v(v_0, a, t):
+    return v_0 + a * t
+
+# if we don't no velocity
+def delta_x(v_0, t, a):
+    return v_0 * t + 0.5 * a * t * t
+
+# if we don't know time
+def v_squared(v_0, a, delta_x):
+    return v_0 * v_0 + 2 * a * delta_x
+
+# if we don't know acceleration
+def delta_x(v, v_0, t):
+    return 0.5 * t * (v + v_0)
+
+# if we don't know initial velocity
+def delta_x(v, t, a):
```

Comment:
Don't think this is right. Where did you get this from?

Reply:
The last one in the file was incorrect, so I removed it, but all the others are fine.

Source: https://www.khanacademy.org/science/physics/one-dimensional-motion/kinematic-formulas/a/what-are-the-kinematic-formulas